.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: docker
+
+# Trigger the workflow when:
```

Comment:
I think this triggering configuration means that Docker images for `stable/*` branches will only be built the first time a new `stable/*` branch is pushed to GitHub and never again, unless there is a manual change to something in the `docker/` directory.

In my opinion, this is fine for now, but perhaps file an issue so that we document this current limitation and perhaps address it in the future, if it will become a frequent issue or [when GitHub Actions gain support for triggering workflows manually](https://github.community/t/github-actions-manual-trigger-approvals/16233) and we can just document how to do that when needed.